src/models/EventPayload.ts

```diff
@@ -10,10 +10,12 @@ export interface EventPayload {
   action: Action;
   /** Unique identifier to tie together events in a single browsing session */
   sessionId?: string;
-  /** The URL of the page where the event occurred */
+  /** The URI of the page where the event occurred */
```

Comment:
Hmn, i think it's not quite right to call it the URI of the page. All URL's are of URI format, so maybe we can keep the word URL and mention that it must be in URI format? 

Thinking about this again it's probably sufficient to just say URL. If you try passing in a non-url you'll get a 4xx error from the events API mentioning the format is wrong so seems intuitive. 

Sorry let's revert this back to URL across the board. LGTM otherwise.